util/httputil/http_util_test.go

```diff
@@ -1,12 +1,16 @@
-package httputil
+package httputil_test
```

Comment:
This is just my own personal preference to ensure testing the public API but again happy to update this back.